src/ios/CodePush.m

```diff
@@ -355,7 +355,7 @@ - (void)pluginInitialize {
         [self handleUnconfirmedInstall:NO];
     }
 
-    [self navigateToLocalDeploymentIfExists];
+    //[self navigateToLocalDeploymentIfExists];
```

Comment:
If this is not needed anymore, we can probably remove it altogether. If there might still be cases where we need to call this, we might need some more changes here.

Reply:
Removed.
I don't think it is necessary to call that method. I tried all the possibilities and that method do nothing